mlir/utils/performance/tuningRunner.py

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env python3
+
+from perfCommonUtils import Operation
+import getopt
+import subprocess
+import sys
+from pathlib import Path
+import argparse
+
+from typing import Optional
+import perfRunner
+from perfRunner import PerfConfiguration
+from perfRunner import ConvConfiguration
+from perfRunner import GemmConfiguration
+from perfRunner import Paths
+
+# global variables.
+BENCHMARKING_RESULT_FILE_NAME = 'results.stats.csv'
+
+#Tune MIOpen with MLIR kernels
+def tuneMLIRKernels(configs, paths: Paths, arch):
```

Comment:
Is there any reason to have this? I'd think we can run convolutions through the same flow as gemms in this script. Comparing against MIOpen can be elsewhere.